lib/dolly/property_manager.rb

```diff
@@ -43,5 +43,12 @@ def assign_identity_properties(opts = {})
       id_presence = opts[:id] || opts[:_id] || opts['id'] || opts['_id']
       self.id = id_presence if id_presence
     end
+
+    def read_property name
```

Comment:
where is this method being used?

Reply:
not used, the change should be done in the apps to use the correct method, i'll remove this